### DISCUSSION TITLE
How to combine the training labels from different jobs to have one SGP model?

### DISCUSSION THREAD

**Discussion by TSMtang** [mir-group/flare#390] [Q&A]
Dear Yu Xie,

In your paper on H/Pt system, I noticed that you finished your training with 4 jobs: H2, Pt bulk, Pt surface, PtH. You collected the training labels from these jobs and combined them into one SGP model. 

Now I'm also trying to do similar things. Your 4 different training jobs are clear but there is no example of combining the training labels into a new SGP model. I wonder if there is any example available or at least I can get some ideas from you. 

I found out that all the local environment information together with all labels is saved in the flare.json file. Is this something I can work on? What would you suggest?

Best regards
Shengming Tang

**Comment by YuuuXie** [comment]
Hi @TSMtang , you can put together all the frames in the 4 training jobs, (their dft output, or .xyz files, or extract from the flare output file), can concatenate them into an .xyz file, and use our offline training method with `FakeMD`
